docs/spark-queries.md

```diff
@@ -357,6 +381,31 @@ SELECT * FROM prod.db.table.all_data_files;
 |      0|s3://.../dt=20210103/00000-0-26222098-032f-472b-8ea5-651a55b21210-00001.parquet|    PARQUET|{20210103}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210103}|{1 -> 3, 2 -> 20210103}|        null|          [4]|        null|            0|
 |      0|s3://.../dt=20210104/00000-0-a3bb1927-88eb-4f1c-bc6e-19076b0d952e-00001.parquet|    PARQUET|{20210104}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210104}|{1 -> 3, 2 -> 20210104}|        null|          [4]|        null|            0|
 
+#### All Delete Files
+
+To show all the table's delete files and each file's metadata:
```

Comment:
same comment for "all the table's" 

do we need each file's metadata?

Reply:
followed the same convention of previous tables like `files`

Reply:
ok. The existing convention is still confusing for me to read. But we can optimize in a follow up. Ok for me to keep it similar now.